### PR TITLE
siminar 11

### PR DESCRIPTION
**Pull request by neilzwh**
seminar 11 update

### CODE REVIEW

File: docs/_posts/2021-12-10-seminar-minutes-2021-11.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Bot Friday 2021年第11次活动纪要
+author: neilzwh
```
Comment (lijiarui) [inline docs/_posts/2021-12-10-seminar-minutes-2021-11.md:3]: In order to show your profile in the blog, please add your profile in this file: https://github.com/wechaty/bot5/blob/main/docs/_data/authors.yml

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for your post! Please change as I request then we would merge this PR.

File: docs/_posts/2021-12-10-seminar-minutes-2021-11.md (hunk 1 above)
Comment (neilzwh) [reply docs/_posts/2021-12-10-seminar-minutes-2021-11.md:3]: Already update author information. Thanks.

**Review by lijiarui — APPROVED** [review]
Cheers!

**Pull request merged**